content/swagger/resources/products/definitions/product.yaml

```
@@ -53,7 +53,15 @@ properties:
               description: Product value
             linked_data:
               type: object
-              description: Object containing labels of attribute options (only when query parameter "with_attribute_options" is set to "true")
+              x-read-only: true
+              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true")
```

```suggestion
              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true"). See <a href='/concepts/products.html#the-linked-data-format'>the `linked_data` format</a> section for more details.
```
Link to check, I'm not sure.

---

See <a href='/concepts/products.html#the-linked_data-format'>the `linked_data` format</a> section for more details.
```suggestion
              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true"). See <a href='/concepts/products.html#the-linked_data-format'>the `linked_data` format</a> section for more details.
```

Testé en local ;)